packages/manager/modules/pci/src/components/project/flavors-list/flavor.class.js

```diff
@@ -47,4 +48,10 @@ export default class Flavors {
   hasOsType(os) {
     return this.osType === os;
   }
+
+  isIOPS() {
+    return find(CATEGORIES, ({ pattern }) => {
+      return pattern.test(this.type);
+    });
+  }
```

Comment:
I'm having trouble understanding how this can say if an instance is IOPS or not. If it's a gpu instance it will return true as well, right ?

Reply:
As i discuss with Maxime HURTREL we target only IOPS category. Which is contains only the I1 instances

Reply:
Ok, but here you don't test IOPS category, here you are trying to find a compatible category.
As the method is called `isIOPS`, it should returns a boolean.

Reply:
If I have a look to category constants https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.constants.js#L3
I see it's an array containing all categories, so if I create an instance with type `gpu`, this will return 
`
  {
    category: 'accelerated',
    title: 'GPU',
    pattern: /gpu|nvme/,
  },
` and it's a truthy value.
Also looks like the function returns an object instead of a boolean so I think it's missing a check like 
```suggestion
  isIOPS() {
    return find(CATEGORIES, ({ pattern }) => {
      return pattern.test(this.type);
    }).title === 'IOPS'
  }
```

Reply:
Or search the IOPS category and test a RegExp only one time ;)
```suggestion
  isIOPS() {
    const { pattern } = find(CATEGORIES, ({ title }) => title === 'IOPS');
    return pattern.test(this.type);
  }
```

Reply:
Done